app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionDetailsFragment.kt

```diff
@@ -46,17 +46,21 @@ class SuggestedEditsContributionDetailsFragment : Fragment() {
     }
 
     private fun updateTopGradient() {
+        val color: Int
+        val headerColor: Int
         if (contribution.sizeDiff < 0) {
-            topView.background = GradientUtil.getPowerGradient(R.color.red90, Gravity.TOP)
-            contributionDiffIndicatorLine.setBackgroundColor(ResourceUtil.getThemedColor(requireContext(), R.attr.colorError))
-            contributionDiffText.setTextColor(ResourceUtil.getThemedColor(requireContext(), R.attr.colorError))
-            (requireActivity() as SuggestedEditsContributionDetailsActivity).updateStatusBarColor(ContextCompat.getColor(requireActivity(), R.color.red90))
+            color = ResourceUtil.getThemedColor(requireContext(), R.attr.colorError)
+            headerColor = ColorUtils.compositeColors((if (WikipediaApp.getInstance().currentTheme.isDark) 0x4c000000 else 0x1c000000) or (color and 0xffffff),
```

Comment:
Isnt it better to have a static color declared for these transparencies in the styles? Since we have all the information at compile time... the alphas arent changing dynamically?